main.js

```diff
@@ -1274,19 +1274,16 @@ addStealExtension(function (loader) {
 		}
 	};
 
-if(typeof System !== "undefined") {
-	addEnv(System);
-}
-
-function addEnv(loader){
-	// Add the extension to _extensions so that it can be cloned.
-	loader._extensions.push(addEnv);
+// Steal Env Extension
+// adds some special environment functions to the loader
+addStealExtension(function (loader) {
```

Comment:
Ok, no problem. I usually always run the grunt task when sending a PR. It's not an ideal system but I can't think of a better way right now.